### ISSUE TITLE
Add option to log (logical) epoch and other stats 

### ISSUE THREAD

**Issue by emjotde** [marian-nmt/marian-tensorboard#9]
If possible, it would be great if:
* the x-axis could have epoch-based displays instead of only step (step size changes often during training)
*  we could also log more stats like:
   * gradient norm (gNorm)
   * last batch size, that would be the `M` in `Cost X * N @ M` 
   * Speed in words/s
